### PR TITLE
Embryo/Larva chestburst changes

### PR DESCRIPTION
**Pull request by kugamo**
## About The Pull Request

This PR removes the sounds announcing a larva has been occupied within the host that is about to chestburts, changing it so it only plays for the larva's client. Instead the host will start screaming uncontrollably as the larva digs its way out D: Lastly I added unique sprites for the embryo instead of them just using the dead larva sprite, as well as a unique hud icon for the chest burst stage.

## Why It's Good For The Game

Ethereal noises for when someone about to chest burst is LRP and equates to an OOC alarm, screaming and being paralyzed from pain is HRP and a good IC way of telegraphing this. Sprites for underdeveloped alien embryo to distinguish it from a fully developed larva are good.

Aseprite_HDygYgnDVK

dreammaker_yQCBa50UNm

## Changelog

:cl:
add: Hosts about to chest burst wont play an ethereal sound to everyone, and instead will scream uncontrollably. 
imageadd: Alien and Xenoid embryos now have unique sprites!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 1]
```diff
@@ -92,6 +92,9 @@
 			if(iscarbon(affected_mob))
 				var/mob/living/carbon/C = affected_mob
 				C.med_hud_set_status()
+	for(var/i in affected_mob.effects_list)
+		if(istype(i, /datum/effects/pain))
+			message_admins("pain: -->[i]")
```
Comment (carlarctg) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm:97]: debug comment

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 2]
```diff
@@ -111,15 +114,19 @@
 		if(4)
 			if(prob(1))
 				if(affected_mob.knocked_out < 1)
+					affected_mob.pain.apply_pain(PAIN_BONE_BREAK)
```
Comment (carlarctg) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm:117]: i'm fairly sure this is a misuse of that define, you should probably add a PAIN_CHESTBURSTING define

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 3]
```diff
@@ -111,15 +114,19 @@
...
 					affected_mob.visible_message(SPAN_DANGER("\The [affected_mob] starts shaking uncontrollably!"), \
 												 SPAN_DANGER("You start shaking uncontrollably!"))
 					affected_mob.apply_effect(10, PARALYZE)
 					affected_mob.make_jittery(105)
 					affected_mob.take_limb_damage(1)
 			if(prob(2))
+				affected_mob.pain.apply_pain(PAIN_BONE_BREAK)
 				var/message = pick("Your chest hurts badly", "It becomes difficult to breathe", "Your heart starts beating rapidly, and each beat is painful")
```
Comment (carlarctg) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm:125]: these messages are kind of underwhelming compared to the pain this stage is apparently putting them through

Comment (kugamo) [reply code/modules/mob/living/carbon/xenomorph/Embryo.dm:125]: the `apply_pain` brings them up from nothing to mild, and not even always that (pain code). It stays in these states until the larva gets occupied and chestbursting begins which brings the pain on for real. 
image

**Review by monkeysfist101 — APPROVED** [review]
Sprites Approved

**Review by monkeysfist101 — APPROVED** [review]

**Comment by ThePiachu** [comment]
I tied a few times and I couldn't trigger infected5 hud to show:

image

Could you make sure it's working correctly?

**Review by ThePiachu — APPROVED** [review]
Looks good!

**Pull request merged**